src/utils/sformat.fsi

```diff
@@ -53,6 +53,13 @@ namespace Microsoft.FSharp.Text.StructuredFormat
         | Unbreakable
         | Breakable of int
         | Broken of int
+
+    [<NoEquality; NoComparison>]
+#if COMPILER
+    type internal BoxRange = BoxRange of obj
```

Comment:
That's not a great excuse. Move the range definition, if that's not possible then make the module mutually recursive (I assume they're in the same file?) If they aren't in the same file then create a new file for range.